src/lib/wallet.js

```diff
@@ -102,6 +148,18 @@ export default {
     const account = MemoryAccount({ keypair });
     try {
       const acceptCb = (_, { accept }) => accept();
+
+      let tokenInfo = await Backend.getTokenInfo();
+
+      tokenInfo = Object.entries(tokenInfo).map(token => ({
```

Comment:
why the need to transform, without the info per contact can be accessed "cheaper"